endpointmanager/pkg/endpointmanager/postgresql/fhirendpointinfostore.go

```diff
@@ -255,6 +249,12 @@ func (s *Store) UpdateFHIREndpointInfo(ctx context.Context, e *endpointmanager.F
 	var err error
 	var capabilityStatementJSON []byte
 
+	metadataID, err := s.AddFHIREndpointMetadata(ctx, e.Metadata)
```

Comment:
The reason I made AddFHIREndpointMetadata a part of adding or updating fhir_endpoints_info is because the metadata table would always be updated during an update or add to the fhir_endpoints_info table, and also since the metadata_id is needed to be added to the fhir_endpoint_info table and this id is acquired from AddFHIREndpointMetadata, I thought it made more sense to put these dependent functions together, rather than what I felt might be complicating things by making another completely separate function handle all of this information swapping. But if you both decide you would rather me change this, let me know!

Reply:
Yeah I think that's valid. I feel like if Lizzie were here she would insist that they get separated, but since they are always getting updated together, and it seems unlikely that we would ever go back to separating out the response stuff and the capability statement stuff as two separate queries, I'm fine with leaving them together. I'd still definitely want to hear Matt's thoughts though!